Resources/Prototypes/_StarLight/Roles/Jobs/Security/brigmedic.yml

```diff
@@ -0,0 +1,51 @@
+- type: job
+  id: Brigmedic
+  name: job-name-brigmedic
+  description: job-description-brigmedic
+  playTimeTracker: JobBrigmedic
+  requirements:
+  - !type:DepartmentTimeRequirement
+    department: Security
+    time: 72000 #20 hrs
+  - !type:DepartmentTimeRequirement
+    department: Medical
+    time: 54000 #15 hrs
+  startingGear: BrigmedicGear
+  icon: "JobIconBrigmedic"
+  supervisors: job-supervisors-hos
+  canBeAntag: false
+  access:
+  - Security
+  - Brig
+  - Maintenance
+  - Service
+  - Medical
+  - Cryogenics
+  special:
+  - !type:AddImplantSpecial
+    implants: [ MindShieldImplant ]
+
+- type: startingGear
+  id: BrigmedicGear
+  equipment:
+    jumpsuit: ClothingUniformJumpsuitBrigmedic
+    eyes: ClothingEyesHudMedSec
+    head: ClothingHeadHatBeretBrigmedic
+    id: BrigmedicPDA
+    ears: ClothingHeadsetBrigmedic
+    outerClothing: ClothingOuterCoatAMG
+    neck: ClothingNeckTieRed
+    pocket1: WeaponPistolMk58
+    shoes: ClothingShoesBootsCombatFilled
+    belt: ClothingBeltMedicalFilled
+    back: ClothingBackpackBrigmedic
+    mask: ClothingMaskBreathMedicalSecurity
+    gloves: ClothingHandsGlovesNitrile
+  storage:
+    back:
+    - BoxSurvivalSecurity
+    - Flash
+    - MagazinePistol
+  innerClothingSkirt: ClothingUniformJumpsuitBrigmedic
+  satchel: ClothingBackpackBrigmedic
+  duffelbag: ClothingBackpackBrigmedic
```

Comment:
linter

Reply:
oof, my apologies. Left that to remind myself to do loadout. Will fix in few hours